R/cut.R

```diff
@@ -0,0 +1,137 @@
+#' Cut a numeric Crunch variable
+#'
+#' `crunch::cut()` is equivalent to `base::cut()` except that it operates on
+#' Crunch variables instead of in-memory variables. The function takes a numeric variable
+#' and recodes it into a new categorical variable based on the breaks arugment. You can either
+#' break the variable into evenly spaced categories by specifying the number of breaks, or specify
+#' a numeric vector which identifies the start and end point of each category. For instance specifying
+#' `breaks = 5` will break the vector into five evenly spaced portions while `breaks = c(1, 5, 10)` will
+#' recode the data into two groups based on whether the numeric vector falls between 1 and 5 or 5 and 10.
+#' @param x A crunch variable of the class NumericVariable
+#' @param breaks Either a numeric vector of two or more unique cut points
+#' or a single number giving the number of intervals into which x is to be cut.
+#' @param labels labels for the levels of the resulting category.
```

Comment:
Indicate that they should be strings and if breaks are specified, there should be one fewer label than break.